<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_cur_temp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_daily_high">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>24hr - High</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_daily_low">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>24hr - Low</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weekly_avg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Weekly - Avg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weekly_high">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Weekly - High</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_low_alarm_set">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_high_alarm_set">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>High Alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cur_ups">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>UPS Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_daily_avg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>24hr - Avg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weekly_low">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Weekly - Low</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_alarm_status">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_alarm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_datetime">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date and Time</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
